lib/togglv8/time_entries.rb

```diff
@@ -87,7 +87,25 @@ def get_time_entries(dates = {})
     def update_time_entries_tags(time_entry_ids, params)
       return if time_entry_ids.nil?
       requireParams(params, ['tags', 'tag_action'])
-      put "time_entries/#{time_entry_ids.join(',')}", { 'time_entry' => params }
+      time_entries = put "time_entries/#{time_entry_ids.join(',')}", { 'time_entry' => params }
+      return time_entries if params['tag_action'] == 'add'
+
+      #see https://github.com/toggl/toggl_api_docs/issues/200
+      time_entries = [time_entries] unless time_entries.is_a? Array
+      time_entries_for_removing_all_tags_ids = []
+      time_entries.map! do |time_entry|
+        unless time_entry['tags'].nil?
+          time_entry['tags'] = time_entry['tags'] - params['tags']
+          time_entries_for_removing_all_tags_ids << time_entry['id'] if time_entry['tags'].empty?
+        end
+        time_entry
```

Comment:
There are two purposes: 
1) To fix API problem we have to find failed entries.
For example, we have 3 entries with tags: ['A', 'B', 'C"], ['A', 'B'] and ['A']
and we want to call update_time_entries_tags to remove ['A', 'B'] tags.
On line 98 we will get: ['C'], [], []. The 2nd and 3rd entries are empty and we collect their IDs on line 99 which can be used later on line 104.
2) update_time_entries_tags method returns updated time entries. 
But we can't just return line 104 because in our example we removing tags only for 2nd and 3rd entries. We will have an array only of 2 elements on this line.
To return a correct result I've used map! to update time_entries variable.